Automattic/newspack-plugin
# fix: merge conflicts w/ `trunk`

### All Submissions:






### Changes proposed in this Pull Request:



Fixes merge conflicts and updates webpack config to allow builds to succeed in both WP 6.5 and 6.6. Note that reCAPTCHA v2 is currently broken in its current implementation when applied to RAS-ACC flows. This can be addressed in future PRs. reCAPTCHA v3 is still working.

### How to test the changes in this Pull Request:

Confirm that `watch` and `build` commands succeed, and that the flows work without reCAPTCHA v2 active.

### Other information:
